rcljava/include/org_ros2_rcljava_client_ClientImpl.h

```diff
@@ -25,9 +25,9 @@ extern "C" {
  * Method:    nativeSendClientRequest
  * Signature: (JJJJJLorg/ros2/rcljava/interfaces/MessageDefinition;)V
  */
-JNIEXPORT void
+JNIEXPORT jlong
 JNICALL Java_org_ros2_rcljava_client_ClientImpl_nativeSendClientRequest(
-  JNIEnv *, jclass, jlong, jlong, jlong, jlong, jlong, jobject);
+  JNIEnv *, jclass, jlong, jlong, jlong, jlong, jobject);
```

Comment:
is this backporting a change from osrf/ros2_java?

if not, it will introduce some unfortunate conflicts

Reply:
based on the commits, I see you're actually backporting some changes.
can you update the description and link the PRs (or commits) that are being ported?

We can better keep track of things that way

Reply:
Yes, I ported a few select changes towards getting CI green for Galactic. Sorry for picking things out of order, but I figured it would be better to get CI green for Galactic before porting other changes.

I've updated the PR description with references to the original changes.

Reply:
Let me know if you think there's a better strategy.

Reply:
The strategy sounds fine